.gitignore

```diff
@@ -8,6 +8,7 @@ deploy/test
 storageos-operator.yaml
 cluster-operator.tar
 pf.log
+.cache
```

Comment:
This is mounted into the container when running go inside container and used to cache go build files.